Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.controller.ts

```diff
@@ -307,6 +328,54 @@
                     }
                 }
 
+                function bindCriticalities() {
+                    
+                    const optionMap = criticalityOptions.reduce((acc, next, _) => {
+                        acc[next.Id] = {
+                            text: next.Name,
+                            id: next.Id,
+                            optionalObjectContext: {
+                                id: next.Id,
+                                name: next.Name,
+                                description: next.Description
+                            }
+                        };
+                        return acc;
+                    }, {});
+
+                    //If selected state is expired, add it for presentation reasons
+                    let existingChoice = null;
+                    if ($scope.contract.criticalityTypeId !== undefined && $scope.contract.criticalityTypeId !== null) {
+                        existingChoice = {
+                            id: $scope.contract.criticalityTypeId,
+                            name: $scope.contract.criticalityTypeName
+                        };
+
+                        if (!optionMap[$scope.contract.criticalityTypeId]) {
+                            optionMap[existingChoice.id] = {
+                                text: existingChoice.name,
```

Comment:
you are missing something here
![image](https://user-images.githubusercontent.com/13116889/175810327-b7ff58ea-4e8d-40f6-8b4b-453f6d1524c5.png)